### PR TITLE
Introduce server views to support better server errors

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
Introduce views based on Pug [1] for server errors. Solves #3925

**Previews** 

image

image

**Features**

- Customisable error title & error message
- Optional location and locationMessage

[1] https://pugjs.org/api/getting-started.html

### CODE REVIEW

**Review by christian-kreuzberger-dtx — APPROVED** [review]
Looks good to me.
Pinging @johannes-b and @ermin-muratovic on this so that they are aware of the change.

File: bridge/server/views/error.pug [hunk 1]
```diff
@@ -0,0 +1,18 @@
+html
+  head
+    link(rel="stylesheet", href="/static/error.v1.css")
```
Comment (Kavindu-Dodan) [inline bridge/server/views/error.pug:3]: Need to update relative paths as we could have different deployment base paths

**Review by christian-kreuzberger-dtx — APPROVED** [review]
Approving again :)
LGTM

**Review by ermin-muratovic — APPROVED** [review]
lgtm

**Pull request merged**